template/Makefile.j2

```diff
@@ -31,6 +31,12 @@ docker-build:
 docker-publish:
 	echo "${NEXUS_PASSWORD}" | docker login --username github --password-stdin "${DOCKER_REPO}"
 	docker push --all-tags "${DOCKER_REPO}/${ORGANIZATION}/${OPERATOR_NAME}"
+	REPO_ARTIFACT_BY_DIGEST=$$(docker inspect --format='{{range .RepoDigests}}{{ . }}{{end}}' '${DOCKER_REPO}/${ORGANIZATION}/${OPERATOR_NAME}:${VERSION}' | grep -E '^${DOCKER_REPO}/${ORGANIZATION}/${OPERATOR_NAME}@sha256:[0-9a-f]{64}$$' | head -n1);\
```

Comment:
This could use a comment on what is happening here and why and....where does the pushing happen?
Is that `cosign sign`?

Reply:
Yes, `cosign sign` automatically pushes the signature. Added some comments :slightly_smiling_face: